content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -182,12 +248,16 @@ token:          ...
 ```
 
 {{< note >}}
-The content of `token` is elided here.
+By design, `kubectl describe` hides the confidential data in a Secret.
```

Comment:
revert this line, `kubectl describe` specifically shows service account token content